app/graphql/resolvers/submissions_resolver.rb

```diff
@@ -91,11 +91,13 @@ def not_allowed_ids?
     return false if admin? || !@arguments.key?(:ids)
     return true if partner?
 
-    current_user = User.find_by(gravity_user_id: @context[:current_user])
+    # Because there can by more than one user for the currenty user in the DB
+    # TODO: Remove duplicates
+    current_users = User.where(gravity_user_id: @context[:current_user])
 
-    base_submissions.select(:user_id).distinct.map { |s| s.user_id } != [
-      current_user&.id
-    ]
+    base_submissions.select(:user_id).distinct.select do |submission|
+      !current_users.pluck(:id).include?(submission.user_id)
```

Comment:
*Non-blocking*: It may be more performant if we don't have to call `pluck` on `current_users` for each submission.
```suggestion
    current_users_ids = current_users.pluck(:id)
    base_submissions.select(:user_id).distinct.select do |submission|
      !current_users_ids.include?(submission.user_id)
```

Reply:
Good point! I'll update the code in the case we have to keep this workaround for a longer time.